skill_tagging/static/tagging.js

```diff
@@ -0,0 +1,68 @@
+const tagSkillContainer = document.getElementById("tag-verification-tags-container-id");
+var tagSkillVerifiedTags = [];
+var tagSkillIgnoredTags = [];
+
+function tagVerificationSetNoneToFalse(source) {
+  if (!source.checked) {
+    return;
+  }
+  checkbox = document.getElementById("tagVerificationUnselectAllId");
+  checkbox.checked = false;
+}
+
+function tagVerificationVerifyTags(url) {
+  var csrf_token = document.cookie.split(";").find(c => c.startsWith("csrftoken="))?.split("=")[1];
+  var checkboxes = document.getElementsByName('tag-verification-skills');
+  var noneSelectedCheckbox = document.getElementById('tagVerificationUnselectAllId');
+  // clear containers
+  tagSkillVerifiedTags = [];
+  tagSkillIgnoredTags = [];
+  for (var i = 0, n = checkboxes.length; i < n; i++) {
+    if (checkboxes[i].checked) {
+      tagSkillVerifiedTags.push(parseInt(checkboxes[i].value));
+    } else {
+      tagSkillIgnoredTags.push(parseInt(checkboxes[i].value));
+    }
+  }
+  if (tagSkillVerifiedTags.length === 0 && !noneSelectedCheckbox.checked) {
+    alert("Please select atleast one skill or check 'None of these subjects were covered' checkbox!");
```

Comment:
Not a blocker, but would it be better to just disable the submit button until at least one of the options is selected, instead of displaying an alert?

Reply:
@pkulkark Since it is pure javascript, I was trying to keep it simple. But it turned out to be pretty simple to implement, so added it. :+1:

Reply:
Thanks @navinkarkera. Can we get rid of this alert then?